pkg/config/config.go

```diff
@@ -90,6 +90,7 @@ type MySQLConfig struct {
 	ReaderHostname           string `mapstructure:"readerHostname"`
 	ReaderMaxIdleConnections int    `mapstructure:"readerMaxIdleConnections"`
 	ReaderMaxOpenConnections int    `mapstructure:"readerMaxOpenConnections"`
+	DSN                      string `mapstructure:"dsn"`
```

Comment:
We’ll need to add a separate `ReaderDSN` as well which is a DSN for the reader endpoint (similar to how we have `ReaderHostname`)